setup.cfg

```diff
@@ -128,6 +128,7 @@ all =
     apache-airflow-providers-http
     apache-airflow-providers-snowflake
     apache-airflow-providers-sftp
+    # TODO: Increment microsoft-azure version as per the upcoming release for deprecating WasbPrefixSensorAsync
```

Comment:
This needs to be taken care of by updating the corresponding upcoming release version that includes the Airflow OSS provider PR https://github.com/apache/airflow/pull/36806 is released